Source/WebKit/NetworkProcess/storage/FileSystemStorageHandle.cpp

```diff
@@ -41,6 +41,7 @@ constexpr char pathSeparator = '/';
 constexpr uint64_t defaultInitialCapacity = 1 * MB;
 constexpr uint64_t defaultMaxCapacityForExponentialGrowth = 256 * MB;
 constexpr uint64_t defaultCapacityStep = 128 * MB;
+constexpr uint64_t defaultMaxCapacity = 16 * GB;
```

Comment:
Why 16GB?
It seems we convert from uint64_t to int64_t when calling FileSystem::seekFile, so the max size could be std::numeric_limits<int64_t>::max().

Reply:
Just to avoid file getting too big, as we currently don't have an upper limit of quota (i.e. WebKit client could allow quota to grow unboundedly, like our TestRunner).

Reply:
I could drop this check if you think it's fine to allow std::numeric_limits<int64_t>::max().